files/fr/web/web_components/using_templates_and_slots/index.html

```diff
@@ -1,42 +1,41 @@
 ---
-title: Utilisation des templates et des slots
+title: Utiliser template et slot
```

Comment:
I'd suggest to clarify that it's about the template and slot **elements**, because the heading sounds kinda weird as it :)
> Utiliser les éléments template et slot